src/main/scala/Url.scala

```diff
@@ -1,3 +1,98 @@
-object Url {
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import akka.http.scaladsl.model.{ContentTypes, HttpEntity, StatusCodes}
+import akka.http.scaladsl.server.Directives._
+import akka.util.Timeout
+import cats.effect.IO
 
+import scala.concurrent.duration._
+import java.sql.Timestamp
+import scala.language.postfixOps
+import spray.json._
+import doobie._
+import doobie.implicits._
+import doobie.implicits.javasql._
+
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+
+case class UrlIn(userId: Int, value: String, timestamp: String)
+case class UrlOut(userId: Int, value: String, timestamp: Timestamp)
```

Comment:
Why do you need different classes? (a potential hint against it can be the fact that they have the same fields).